src/framework/components/render/system.js

```diff
@@ -12,6 +12,7 @@ const _schema = [
 // order matters here
 const _properties = [
     'material',
+    'meshInstances',
```

Comment:
Should `renderStyle` be here too?

Reply:
done now, add to where it needs to be for cloning to work, thanks!